scr/hw6/Main.java

```diff
@@ -0,0 +1,32 @@
+package hw6;
+
+import java.util.Random;
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+        Scanner s = new Scanner(System.in);
+        String[][] arr = new String[4][4];
+
+        Random a = new Random();
+
+        for (int i = 0; i < arr.length; i++) {
+            for (int j = 0; j < arr[i].length; j++) {
+                arr[i][j] = Integer.toString(a.nextInt(10));
+                //arr[i][j] = s.next();
+            }
+        }
+       /* for(int i=0;i<arr.length;i++)
```

Comment:
Clean-up

Reply:
I left comments because there were several solutions, and so that the array could be seen, if necessary